src/components/tutorialEditor/stepContent.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from "reactn";
+import styled, { AnyStyledComponent } from "styled-components";
+import "../app/normalize.css";
+
+import AceEditor from "react-ace";
+import { Icon } from "@iconify/react";
+import chevronUp from "@iconify-icons/feather/chevron-up";
+import chevronDown from "@iconify-icons/feather/chevron-down";
+import bxTrashAlt from "@iconify-icons/bx/bx-trash-alt";
+
+import "ace-builds";
+import "ace-builds/webpack-resolver";
+import "ace-builds/src-noconflict/mode-python";
+import "ace-builds/src-noconflict/ext-language_tools";
+import "ace-builds/src-noconflict/theme-tomorrow_night_eighties";
+
+import { background, grey } from "../../constants";
+import { Button } from "../textEditor/button";
+
+interface DeleteProps {
+  visible: string;
+}
+
+interface ContentBlock {
+  type: string;
+  value: string;
+}
+
+const ContentBlock: AnyStyledComponent = styled.div`
+  display: flex;
+  border: 1px solid ${grey};
+  margin-bottom: 20px;
+`;
+
+const ContentBlockInnerWrapper: AnyStyledComponent = styled.div`
+  padding: 20px;
+  width: ${(props: DeleteProps): string =>
+    props.visible === "flex" ? "93%" : "100%"};
+`;
+
+const UpDownContainer: AnyStyledComponent = styled.div`
+  position: relative;
+  float: right;
+  display: inline-block;
+  margin-bottom: 15px;
+`;
+
+const Arrow: AnyStyledComponent = styled(Icon)`
+  width: 40px;
+  height: 40px;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+const TextInput: AnyStyledComponent = styled.textarea`
+  width: 100%;
+  min-height: 82px;
+  background-color: #46464e;
+  outline: none;
+  border: none;
+  border-radius: 1px;
+  padding: 5px;
+  color: #f5f5f5;
+  resize: none;
+`;
+
+const DeleteBlock: AnyStyledComponent = styled.div`
+  display: ${(props: DeleteProps): string => props.visible};
+  border-left: solid 1px #727272;
+  width: 7%;
+  text-align: center;
+  justify-content: center;
+  &:hover {
+    cursor: pointer;
+    background: rgba(255, 255, 255, 0.1);
+  }
+`;
+
+const TrashIcon: AnyStyledComponent = styled(Icon)`
+  align-self: center;
+  width: 22px;
+  height: 22px;
+`;
+
+interface ContentBlock {
+  type: string;
+  value: string;
+}
+
+interface StepContentProps {
+  value: ContentBlock;
+  index: number;
+  content: ContentBlock[];
+  setContent: Function;
+}
+
+const StepContent: React.FC<StepContentProps> = (props): JSX.Element => {
+  const { value, index, content, setContent } = props;
+
+  const [hovering, setHovering] = useState(-1);
+
+  const codeInput = {
```

Comment:
```suggestion
  const codeInput: CSSProperties = {
```
Also TypeScript didn't like it untyped. I think it's because TypeScript can't tell the different between a regular JavaScript object and a JavaScript object that represents CSS styles, hence the `CSSProperties` type.